CHANGES/573.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix misleading URLs in the documentation.
```

Comment:
This is not a commit message. It shouldn't be in imperative mood, but rather in the past tense or sometimes present tense with structures like "it is now this way". The idea is that it should imply a comparison with the previous release, while targeting the end users as the audience, not contributors/developers: https://frozenlist.aio-libs.org/en/latest/contributing/guidelines/#adding-change-notes-with-your-prs.

Reply:
Thanks for the guidance! I've replaced this news fragment.